boards/silabs/radio_boards/siwx917_rb4338a/doc/index.rst

```diff
@@ -0,0 +1,116 @@
+.. zephyr:board:: siwx917_rb4338a
+
+
+Overview
+********
+
+The SiWx917-RB4338A (aka BRD4338A) radio board provides support for the Silicon
+Labs SiWG917 SoC. This board cannot be used stand-alone and requires a a
+`Wireless Pro Kit`_ Mainboard (Si-MB4002A aka BRD4002A), for power, debug
+options etc.
+
+This board might be sold as a part of the `SiWx917-PK6031A`_ bundle ("SiWx917
+Wi-Fi 6 and Bluetooth LE 8 MB Flash Pro Kit"), which includes the BRD4002A
+Mainboard in addition of the BRD4338A.
+
+SiWG917 is an ultra-low power SoC that includes hardware support for Single-Band
+Wi-Fi 6 + Bluetooth LE 5.4, Matter...
+
+Hardware
+********
+
+For more information about the SiWG917 SoC and BRD4338A board, refer to these
+documents:
+
+- `SiWG917 Website`_
+- `SiWG917 Datasheet`_
+- `SiWG917 Reference Manual`_
+- `BRD4338A Website`_
+- `BRD4338A User Guide`_
+
+
+Supported Features
+==================
+
+The ``siwx917_rb4338a`` board target supports the following hardware features:
+
++-----------+------------+------------------------+
+| Interface | Controller | Driver/Component       |
++===========+============+========================+
+| CMU       | on-chip    | clock control          |
++-----------+------------+------------------------+
+| MPU       | on-chip    | memory protection unit |
++-----------+------------+------------------------+
+| NVIC      | on-chip    | interrupt controller   |
++-----------+------------+------------------------+
+| UART      | on-chip    | serial                 |
++-----------+------------+------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Applications for the ``siwx917_rb4338`` board target can be built in the usual
```

Comment:
Missing `a` from board name here, and consistently in the rest of the Flashing section.